### PR TITLE
126 amplify child theme

### PR DESCRIPTION
**Pull request by benlk**
## Changes

This pull request makes the following changes:

- creates `amplify` child theme:
    - stylesheet
    - Grunt
    - gitignore
    - LESS variables and common styles pulled in from https://github.com/INN/inn/blob/master/less/variables.less to match other INN sites' branding
- updates the README in `themes/` to refer to more themes present on this site (see https://github.com/INN/umbrella-inndev/issues/60 for why some of these child themes aren't present in this repo)

## Why

For #126, creating the child theme for the Amplify site.

## Testing/Questions

Features that this PR affects:

- amplify.inn.org, which is currently being bootstrapped
- docs in this repo

Questions that need to be answered before merging:

- [x] Is this PR targeting the correct branch in this repository? I'm setting this to "staging" because I created the branch based off of staging, and because we'll be deploying to staging. It [looks like the current master is ahead of the current staging branch](https://github.com/INN/umbrella-inndev/network), so merging this to master and deploying to prod shouldn't bring unplanned commits into the prod env.
- [ ] is there any other boilerplate we want to include here for #126?

Steps to test this PR:

1. check out this branch
2. in `/wp-admin/network/sites.php`, create a new site, within the inndev multisite, for amplify work on your local env.
3. find the "edit site" view for your new amplify site, and go to the "themes" tab. Enable the "Amplify" theme.
4. Go to your amplify site's dashboard, and activate the "Amplify" theme.

### CODE REVIEW

File: wp-content/themes/amplify/style.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/*
+Theme Name:     Amplify
+Theme URI:      https://github.com/INN/umbrella-inndev/tree/master/wp-content/themes/amplify/
+Description:    Largo child theme.
```
Comment (joshdarby) [inline wp-content/themes/amplify/style.css]: @benlk Can we update this to something more descriptive? Other than that, it looks beautiful!

Comment (benlk) [reply wp-content/themes/amplify/style.css]: Drawing from https://inn.org/project/amplify-news-project/ , perhaps "Amplify News Project" and "Largo child theme for Amplify, with enhanced support for Link Roundups"?

Comment (joshdarby) [reply wp-content/themes/amplify/style.css]: > Largo child theme for Amplify, with enhanced support for Link Roundups

That one sounds better, but up to you 😬

Comment (benlk) [reply wp-content/themes/amplify/style.css]: Updated in 4c9b24b

**Pull request merged**